.circleci/config.yml

```diff
@@ -0,0 +1,105 @@
+# Tagging a commit with [circle front] will build the front page and perform test-doc.
+# Tagging a commit with [circle full] will build everything.
```

Comment:
are these two comments true? doesn't look like it to me from the code

Reply:
I must confess this file is being copy pasted from one project to the other and I am far from fully understanding it (but it works!). I removed the comment.